### PR TITLE
[PLAT-147] Add create checkout session request

### PR DESCRIPTION
**Pull request by wlads**
- feat: implement create checkout session
- chore: include specs
- chore: update readme

Type of Change
--------------

* [ ] Bug fix (non-breaking change which fixes an issue)
* [x] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (would cause existing functionality to not work as expected)
* [ ] This change requires a documentation update
* [ ] This change is in scope for PCI
* [ ] This requires approval from UX
* [ ] This requires approval from Marketing

Links
-----

https://justifi-ai.atlassian.net/browse/PLAT-147

Steps for Testing/QA
--------------------

Checkout branch, run the Gem's console pointing to staging:

```sh
API_STAGING_BASE_URL=https://api.xyz bin/console
```

Setup the SDK using a platform's credentials:

```ruby
Justifi.setup(client_id: "", client_secret: "", environment: "staging")
```

Following the docs: https://developer.justifi.ai/#operation/CreateCheckoutSession

- [x] Create a checkout session `Justifi::CheckoutSession.create(..)`

(tips in the readme: https://github.com/justifi-tech/justifi-ruby/blob/66224b8576e3ae2bd5d661a8eac54dc17fe82ac7/README.md#create-a-checkout-session)

### CODE REVIEW

**Comment by krististepanek** [comment]
In order to get the hosted checkout session to actually work after creating it through the gem, I had to use the url `#{hosted_checkout_url}/r/#{checkout_session_id}`
Do we need to update that in the readme? 
```
# use the session id to access your hosted checkout page at:
hosted_checkout_url = "https://hosted-checkout.justifi.ai/r/#{session.data.checkout_session_id}"
```

**Review by krististepanek — APPROVED** [review]
📨

File: spec/stubs.rb [hunk 1]
```diff
@@ -1,5 +1,3 @@
-require "securerandom"
```
Comment (wlads) [inline spec/stubs.rb:1]: Moved this from here to the main lib because @krististepanek had some issue running payment intent create

**Review by krististepanek — APPROVED** [review]
🪀

**Pull request merged**